enioka_scan/src/main/AndroidManifest.xml

```diff
@@ -98,6 +98,16 @@
             </intent-filter>
         </service>
 
+        <service
+            android:name=".sdk.mock.MockProvider"
+            android:description="@string/service_description"
+            android:exported="false">
+
+            <intent-filter>
```

Comment:
Shouldn't we add this provider to excludedProviderKeys in the default search options? Otherwise it will be found in real production runs.

Reply:
The other solution is to create it inside a different module that is imported as a test scope...

Reply:
Went for the second option, Mock sdk is now in a separate "enioka_scan_mock" project similar to non-OSS sdks and can be included or not on its own.